src/sidebar/services/frame-sync.js

```diff
@@ -387,6 +399,17 @@ export class FrameSyncService {
     // Synchronize highlight visibility in this guest with the sidebar's controls.
     guestRPC.call('setHighlightsVisible', this._highlightsVisible);
     guestRPC.call('featureFlagsUpdated', this._store.profile().features);
+
+    // If we have content banner data, send it to the guest. If there are
```

Comment:
Is calling this "banner data" too specific here?

Reply:
You mean if we decide to make the UI something other than a banner?